Source/Core/Core/NetPlayClient.cpp

```diff
@@ -611,6 +613,9 @@ unsigned int NetPlayClient::OnData(sf::Packet& packet)
       packet >> m_net_settings.m_SyncCodes;
       packet >> m_net_settings.m_SyncAllWiiSaves;
 
+      for (size_t i = 0; i < 4; i++)
```

Comment:
After making m_WiimoteExtension an std::array, this can just be `for (int& extension : m_WiimoteExtension)`, avoiding the need for a magic number.